From 3b81667a19be2767d21ac0d50c79c678b8e923e9 Mon Sep 17 00:00:00 2001
From: Andre Meyering <info@andremeyering.de>
Date: Mon, 31 May 2021 10:27:08 +0200
Subject: [PATCH] [QMake] Fix include path for internal QuaZip dependency

Commit 6f9d61569750fd6e3a5dd78985fea20e069c1a1d forgot to adapt the
include path.  On some systems, it still worked, on others it failed.
---
 MediaElch.pro | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/MediaElch.pro b/MediaElch.pro
index 6944a1750..2a86745d2 100644
--- a/MediaElch.pro
+++ b/MediaElch.pro
@@ -15,6 +15,8 @@ contains(CONFIG, USE_EXTERN_QUAZIP) {
     DEFINES += QUAZIP_BUILD
     DEFINES += QUAZIP_STATIC # Required by Quazip to export symbols
     include(third_party/quazip/quazip/quazip.pri)
+    # For correct include paths
+    INCLUDEPATH += third_party/quazip
 }
 
 TEMPLATE = app
